source/MaterialXFormat/XmlExport.cpp

```diff
@@ -0,0 +1,105 @@
+//
+// TM & (c) 2021 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <MaterialXFormat/XmlExport.h>
+
+#include <MaterialXFormat/Util.h>
+
+MATERIALX_NAMESPACE_BEGIN
+
+//
+// XmlExportOptions methods
+//
+
+XmlExportOptions::XmlExportOptions() :
```

Comment:
What is the semantic difference between XmlExport and XmlWrite?  Could this instead be implemented as an extension of the XmlWrite system, rather than a separate parallel system?

Reply:
There are a few additional options, I need to consolidate with RTMaterialX. I'll remove this for now and reintroduce in a separate PR